packages/back-end/src/api/experiments/postExperiment.ts

```diff
@@ -47,20 +47,26 @@ export const postExperiment = createApiRequestHandler(postExperimentValidator)(
       );
     }
 
-    const user = await getUserByEmail(owner);
+    const ownerId = await (async () => {
+      if (!onwerEmail) return req.context.userId;
```

Comment:
The original feature request specification is:
> if the auth was done via a personal access token, and an owner property wasn't provided, we should default to the owner of the personal access token.

Is that the right interpretation?

Reply:
Yes, that's correct